src/content.js

```diff
@@ -116,8 +143,19 @@ module.exports = function (req, res) {
     async.apply(layout, presented)
   ], function (err, result) {
     if (err) {
-      logger.error("Assembling: " + err);
-      res.status(500).send(page500);
+      var code = 404;
+      if(err.type == "NotFound") {
```

Comment:
:+1: 

Yeah, I looked at the docs and the source and didn't see anything about it.